frontend/src/locales/scripts/en.json5

```
@@ -871,14 +871,14 @@
   },
   audioDetails: {
     genreLabel: "Genre",
-    relatedAudios: "Related audio",
+    relatedAudios: "Related audio track",
```

Please use the plural form.
```suggestion
    relatedAudios: "Related audio tracks",
```